### PR TITLE
🌱Add trigger phrases in the doc to keep VM for e2e CI jobs

### PR DESCRIPTION
**Pull request by smoshiur1237**
This PR will add the trigger phrases to keep VM for e2e job

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,4 +145,22 @@ Upgrade tests:
...
+- **/keep-test-ubuntu-e2e-main** run keep e2e tests with CAPM3 API version v1beta1 and branch main on Ubuntu
+- **/keep-test-centos-e2e-main** run keep e2e tests with CAPM3 API version v1beta1 and branch main on CentOS
+- **/keep-test-ubuntu-e2e-release-1-3** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.3 on Ubuntu
+- **/keep-test-centos-e2e-release-1-3** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.3 on CentOS
+- **/keep-test-ubuntu-e2e-release-1-2** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on Ubuntu
+- **/keep-test-centos-e2e-release-1-2** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.2 on CentOS
+- **/keep-test-ubuntu-e2e-release-1-1** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on Ubuntu
+- **/keep-test-centos-e2e-release-1-1** run keep e2e tests with CAPM3 API version v1beta1 and branch release-1.1 on CentOS
+
+Note: Test VM created with these phrases will not be kept forever but deleted after 24 hours, to avoid garbage collection of VMs.
```
Comment (lentzi90) [inline README.md:164]:
```suggestion
Note: Test VM created with these phrases will not be kept forever but deleted after 24 hours.
```

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by smoshiur1237** [comment]
/hold
The trigger phrases exists in our CI but we need to change the behavior of the e2e, so that after a failed job it will skip the cleaning of the cluster. So this PR will go in together with a separate PR for e2e

**Comment by smoshiur1237** [comment]
Re-based the PR.
/test-ubuntu-integration-main

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by smoshiur1237** [comment]
PR is ready to get in as keep jobs for integration test are ready. I will add the keep trigger phrases for e2e upgrade, once my gerrit credentials are fixed and will push the keep job for e2e upgrde ci run.
/unhold

**Pull request merged**